Shared/Samples/Change map view background/ChangeMapViewBackgroundView.Model.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import SwiftUI
+import ArcGIS
+
+extension ChangeMapViewBackgroundView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with a streets basemap style.
+        let map: Map = {
+            // Initialize tiled layer.
```

Comment:
```suggestion
            // Initializes tiled layer.
```
We try to follow Apple's documenting convention to use 3rd person tense for verbs. Please change in other places as well.